js-api/src/ui/wizard.ts

```diff
@@ -32,7 +32,7 @@ export class Wizard extends Dialog {
   nextButton = this.addButton('>>', () => this.next());
 
   constructor(options?: {title?: string, helpUrl?: string}) {
-    super(api.grok_Dialog(options?.title, options?.helpUrl, true, true));
+    super(api.grok_Dialog(options?.title, options?.helpUrl, !!(options?.title || options?.helpUrl), true));
```

Comment:
we need two of them !! ?

Reply:
Here it's simply a shorter way to convert to boolean